drafts/docker-open-liberty-raspberry-pi.adoc

```diff
@@ -1,60 +1,73 @@
+Download the other folders and files in that directory to build this image so it is stored locally. You will now be able to use this image as a base for your applications. Instructions to build a Docker image that has your application and configuration pre-loaded can be found link:https://github.com/OpenLiberty/ci.docker#building-an-application-image[here]. 
 
-=== Liberty Bikes 
-Liberty bikes is an application consisting of four microservices and each image for the services use the link:https://github.com/OpenLiberty/ci.docker/blob/master/official/19.0.0.9/microProfile2/java11/openj9/Dockerfile[open-liberty:microProfile2-java11] as a base. Edit the `FROM` target in this Dockerfile so that it uses the image that you just built. 
+[source]
+docker build -t OpenLibertyRaspberryPi .
+
+=== Launch a sample application
 
+Try out your image with a sample application. To begin:
 
 [source]
-FROM <YOUR IMAGE NAME>
-RUN cp /opt/ol/wlp/templates/servers/microProfile2/server.xml /config/server.xml
+git clone https://github.com/OpenLiberty/sample-getting-started
+cd sample-getting-started 
 
-Build the microProfile2-java11 image so it is also stored locally and can be used for the application. For example,   
+Replace the contents of the `pom.xml` file in your working directory with the contents of link:https://github.com/OpenLiberty/guide-getting-started/blob/master/finish/pom.xml[this file] so it is configured to include the `liberty-maven-plugin`.
 
-[source]
-git clone https://github.com/OpenLiberty/liberty-bikes.git
-cd liberty-bikes
-nano auth-service/Dockerfile 
+Copy this link:https://github.com/OpenLiberty/guide-getting-started/blob/master/finish/Dockerfile[Dockerfile] in to your working directory and edit the first line so it has your Open Liberty image as its base: 
 
 [source]
-FROM <YOUR MICROPROFILE IMAGE NAME>
-...
-RUN printf 'httpPort=8082\n\
-httpsPort=8482' > /config/bootstrap.properties
-EXPOSE 8082 8482
+----
+FROM OpenLibertyRaspberry Pi
+
+LABEL maintainer="Graham Charters" vendor="IBM" github="https://github.com/WASdev/ci.maven"
+
+COPY --chown=1001:0 src/main/liberty/config/ /config/
+COPY --chown=1001:0 target/*.war /config/apps/
+----
 
-Edit the `FROM` target from the the other three Dockerfiles in the `frontend`, `game-service` and `player-service` files. You can then build all the microservice applications and deploy them to locally running liberty servers.
+If maven is not already installed on your Raspberry Pi, execute the following command: 
 
 [source]
- ./gradlew start frontend:open
+----
+sudo apt install maven
+----
+
+To build the application that is provided and deploy it to Open Liberty: 
 
-To stop all liberty servers, issue the command:
 [source]
- ./gradlew stop
+----
+mvn clean package liberty:run-server
+----
+
+When the server is ready, go to http://localhost:9080
```

Comment:
Say what they should see when they go to the URL.

Then add a summary paragraph (couple of sentences) summarising what they've done.